cmd/manager/main.go

```diff
@@ -293,3 +305,24 @@ func setupWebhooks(mgr manager.Manager) error {
 
 	return nil
 }
+
+func isKubeMetaKind(kind string) bool {
```

Comment:
This list is lifted from here: https://github.com/operator-framework/operator-sdk/blob/v0.15.2/pkg/k8sutil/k8sutil.go#L142-L180